docs/api_walkthrough.rst

```diff
@@ -54,6 +54,9 @@ If it does not exist in the list, it is desired to :ref:`register the token <add
 
 Registering a token
 --------------------
+.. note::
+   For the Raiden Red Eyes release, it will not be possible to register more than one token, due to security reasons and Red Eyes being an alpha release on the mainnet.
+
```

Comment:
Maybe expand on security reasons a bit, e.g. 'to minimise possible loss in the case of bugs' or similar.